index.d.ts

```diff
@@ -0,0 +1,6 @@
+/**
+Make unhandled promise rejections fail loudly instead of the default [silent fail](https://gist.github.com/benjamingr/0237932cee84712951a2).
+
+@param log - Custom logging function to print the rejected promise. Receives the error stack. Default: `console.error`.
```

Comment:
@BendingBender You might be interested in and maybe have some opinion about this issue: https://github.com/Microsoft/tsdoc/issues/151